packages/package-manager/src/CocoaPodsPackageManager.ts

```diff
@@ -47,6 +47,25 @@ export class CocoaPodsPackageManager implements PackageManager {
     return existsSync(path.join(projectRoot, 'Podfile'));
   }
 
+  static isUsingRubyBundler(): boolean {
+    return existsSync('Gemfile');
```

Comment:
Be sure to pass in the projectRoot and join the path components (like we do for `isUsingPods`), if this file is supposed to live in the user home directory, then use `os.homedir()`.